templates/docs/examples/patterns/switch.html

```diff
@@ -9,16 +9,19 @@
     <input type="checkbox" class="p-switch" id="checkbox1" checked/>
     <span class="p-switch__slider"></span>
 </label>
+<br />
```

Comment:
Probably not for this PR, but seems this may be a breaking change for some if they relied on switch being block.

But also, seems like switches are these weirdly structured components - root label element doesn't have class name, main component class name is on input but there is also separate slider element outside of it...
Maybe we should refactor it as part of 3.0 as well while we are at it.